HousingRegisterApi/V1/UseCase/GetInternalReportUseCase.cs

```diff
@@ -18,17 +18,20 @@ public class GetInternalReportUseCase : IGetInternalReportUseCase
         private readonly ILogger<GetInternalReportUseCase> _logger;
         private readonly IFileGateway _fileGateway;
         private readonly IApplicationApiGateway _gateway;
+        private readonly IActivityHistory _activityGateway;
```

Comment:
`IActivityHistoryGateway`

Reply:
i changed it to activitygateway cos its shorter